### DISCUSSION TITLE
Use Ray with Nuxt application

### DISCUSSION THREAD

**Discussion by lumos94** [spatie/ray#872] [Q&A]
Hello,

Can I use ray within a Nuxt application?

**Comment by freekmurze** [comment]
You can via our JavaScript integration: https://spatie.be/docs/ray/v1/installation-in-your-project/javascript

**Reply by lumos94** [reply]
Nuxt is a Vue framework. Shouldn't I use the Vue integration or is it better for Nuxt to use the Javascript one?
